vmm/src/vm.rs

```diff
@@ -360,6 +360,8 @@ struct VmOpsHandler {
     #[cfg(target_arch = "x86_64")]
     io_bus: Arc<Bus>,
     mmio_bus: Arc<Bus>,
+    #[cfg(feature = "sev_snp")]
+    vm: Arc<dyn hypervisor::Vm>,
```

Comment:
Ditto, this can also be removed.

Reply:
Oh yes I forgot about that change, I thought he was okay with using vm_ops. Just need a usecase around where it would be used. I have changed the code to use vm_fd now.